packages/process/src/node/pseudo-pty.ts

```diff
@@ -0,0 +1,53 @@
+/********************************************************************************
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { IPty } from '@theia/node-pty';
+import { Event, Emitter } from '@theia/core';
+
+export class PseudoPty implements IPty {
+
+    readonly pid: number = -1;
+
+    readonly cols: number = -1;
+
+    readonly rows: number = -1;
+
+    readonly process: string = '';
+
+    handleFlowControl: false;
+
+    protected readonly onDataEmitter = new Emitter<string>();
+    readonly onData: Event<string> = this.onDataEmitter.event;
+
+    protected readonly onExitEmitter = new Emitter<{ exitCode: number, signal?: number }>();
```

Comment:
do we need real emitters? we need to dispose them then

Maybe stub events with `Event.None` instead?